conans/client/rest/file_downloader.py

```diff
@@ -120,19 +120,17 @@ def get_total_length():
                 progress.update(read_response(chunk_size)),
                 file_path
             )
-
+            gzip = (response.headers.get("content-encoding") == "gzip")
```

Comment:
Should we also consider other possible compression algorithms?
```suggestion
            is_compressed = (response.headers.get("content-encoding") in {"gzip", "deflate", "br"})
```
You could test them e.g. here: https://httpbin.org/#/Response_formats

As an alternative you could compute the total_length with `len(response.content)` in those cases, ignoring the Content-Length from headers...

Reply:
I'd prefer to just "revert" that regression, and we could open a new ticket for improving this for next releases. These other compressions types haven't been an issue so far (they would have raised exceptions in the past, just by using them), so I think this is ok as-is.